src/components/svg/lineDrawing.ts

```diff
@@ -195,97 +195,107 @@ export const stackedArea = ({
   yPropTop,
   yPropBottom
 }: StackedAreaTypes) => {
-  const uniqXValues = data
-    .map(d => d.data.map(p => datesForUnique(p[xProp])))
-    .reduce((a, b) => a.concat(b), [])
-    .reduce((p, c) => {
-      if (p.indexOf(c) === -1) {
-        p.push(c)
+  const a = performance.now()
```

Comment:
I assume this should be removed before merging?

Reply:
It sure should be!